### PR TITLE
t3c remove perl dependency and references 

### PR DESCRIPTION
**Pull request by jpappa200**
Perl was deprecated and should be removed in this release of traffic control
<hr/>

## Which Traffic Control components are affected by this PR?

- Traffic Control Cache Config (`t3c`, formerly ORT)

## PR submission checklist
- [ ] This PR has tests 
- [ ] This PR has documentation 
- [x] This PR has a CHANGELOG.md entry 
- [ ] This PR **DOES NOT FIX A SERIOUS SECURITY VULNERABILITY** (see [the Apache Software Foundation's security guidelines](https://apache.org/security) for details)

### CODE REVIEW

File: infrastructure/ansible/roles/ats/defaults/main.yml [hunk 1]
```diff
@@ -39,19 +39,15 @@ ort_crontab:
...
-        traffic_ops_ort.pl syncds warn {{ ort_traffic_ops_url }} '{{ ort_traffic_ops_username }}:{{ ort_traffic_ops_password }}' --login_dispersion=35 --dispersion=420 &>/tmp/ort/syncds.log
       {%- endif -%}
   # Perform a check for user requested content invalidations every minute
   reval:
     schedule: '1-19,21-39,41-59 * * * *'
     user: root
     job: >-
-      {%- if ort_version is not defined or ort_version == omit or ort_version >= '6.0.0-11351' -%}
+      {%- if ort_version is not defined or ort_version == omit -%}
         t3c apply --run-mode=revalidate --cache-host-name='{{ (ansible_nodename.split('.')) | first }}' -vv --traffic-ops-url='{{ ort_traffic_ops_url }}' --traffic-ops-user='{{ ort_traffic_ops_username }}' --traffic-ops-password='{{ ort_traffic_ops_password }}' &> /tmp/trafficcontrol-cache-config/reval.log
```
Comment (jhg03a) [inline infrastructure/ansible/roles/ats/defaults/main.yml:51]: The if clauses here can be removed

File: infrastructure/ansible/roles/ats/tasks/ats.yml [hunk 2]
```diff
@@ -67,10 +50,8 @@
   shell:
     executable: "{{ lookup('pipe','command -v bash') }}"
     cmd: >-
-      {%- if ort_version is not defined or ort_version == omit or ort_version >= '6.0.0-11351' -%}
+      {%- if ort_version is not defined or ort_version == omit -%}
         pkill -9 t3c || true; t3c apply --run-mode=badass --git=yes --cache-host-name='{{ (ansible_nodename.split('.')) | first }}' -vv --traffic-ops-url='{{ ort_traffic_ops_url }}' --traffic-ops-user='{{ ort_traffic_ops_username }}' --traffic-ops-password='{{ ort_traffic_ops_password }}' --wait-for-parents=false 2>&1 | tee /tmp/trafficcontrol-cache-config/badass_out; exit ${PIPESTATUS[0]}
```
Comment (jhg03a) [inline infrastructure/ansible/roles/ats/tasks/ats.yml:54]: The if clause can be removed

**Review by jhg03a — CHANGES_REQUESTED** [review]

File: infrastructure/ansible/roles/ats/defaults/main.yml (hunk 1 above)
Comment (jpappa200) [reply infrastructure/ansible/roles/ats/defaults/main.yml]: removed

File: infrastructure/ansible/roles/ats/tasks/ats.yml (hunk 2 above)
Comment (jpappa200) [reply infrastructure/ansible/roles/ats/tasks/ats.yml]: removed

**Review by zrhoffman — APPROVED** [review]
Looks good, and the ansible roles run without errors. Thanks!

**Pull request merged**